stm32mp1.mk

```diff
@@ -0,0 +1,212 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+################################################################################
+override COMPILE_NS_USER   := 32
+override COMPILE_NS_KERNEL := 32
+override COMPILE_S_USER    := 32
+override COMPILE_S_KERNEL  := 32
+
+PLATFORM ?= stm32mp1-157C_DK2
+OPTEE_OS_PLATFORM := $(PLATFORM)
+
+include common.mk
+
+ifeq ($(PLATFORM),stm32mp1-157A_DK1)
+BREXT_FLAVOR		= STM32MP157A-DK1
+STM32MP1_DTS_BASENAME	= stm32mp157a-dk1
+else ifeq ($(PLATFORM),stm32mp1-157C_DK2)
+BREXT_FLAVOR		= STM32MP157C-DK2
+STM32MP1_DTS_BASENAME	= stm32mp157c-dk2
+else ifeq ($(PLATFORM),stm32mp1-157C_EV1)
+BREXT_FLAVOR		= STM32MP157C-EV1
+STM32MP1_DTS_BASENAME	= stm32mp157c-ev1
+else
+$(error Unknown PLATFORM $(PLATFORM))
+endif
+
+################################################################################
+# Binary images names
+################################################################################
+
+ARM_TF_BIN		:= tf-a-$(STM32MP1_DTS_BASENAME).stm32
```

Comment:
s/ARM_TF_BIN/TFA_BIN/

Reply:
Also is it a ST thing to name binaries with ".stm32"? Something like "bin", "elf" or something similar seems more common and neutral.

Reply:
.stm32 is not neutral. This filename extension reflects the binary format that is proprietary: there is a specific binary header in the image file. It is used (and needed) by the authenticated boot scheme used on stm32mp1 platforms.

> s/ARM_TF_BIN/TFA_BIN/

ok.